internal/pkg/daemon/enricher/container.go

```diff
@@ -111,12 +114,13 @@ func (e *Enricher) getContainerInfo(
 		return nil, errors.Wrap(err, "get container info for pods")
 	}
 
-	info, found := containers[containerID]
-	if !found {
-		return nil, errors.Wrap(err, "no container info for container ID")
+	if info, err := e.infoCache.Get(
+		containerID,
+	); !errors.Is(err, ttlcache.ErrNotFound) {
+		return info.(*containerInfo), nil
 	}
 
-	return info, err
+	return nil, errors.New("found nothing")
```

Comment:
`found nothing` isn't a very descriptive error message, we probably want to keep using something along the lines of `"no container info for container ID"`

Reply:
Fixed :+1: